tests/h/form_test.py

```diff
@@ -86,7 +86,8 @@ def test_passes_args_including_renderer_to_form_ctor(self, Form, pyramid_request
             renderer=Any.instance_of(form.Jinja2Renderer),
         )
 
-    def test_adds_feature_client_to_system_context(self, Form, patch, pyramid_request):
+    @pytest.mark.usefixtures("Form")
```

Comment:
As every test in this class uses `Form` it's probably a good case for `autouse=True`